docs/page/configuration.md

```diff
@@ -282,6 +282,49 @@ To fully support rust and pretty printing of strings when debugging, remember to
 
 ## Go
 
+1.  Installation
+
+      - install gopls
+
+      - Install the delve command by following instructions on [delve -
+        installation](https://github.com/go-delve/delve/tree/master/Documentation/installation).
+
+      - install lsp-mode
+
+      - Put in your emacs configuration.
+
+        ``` elisp
+        (require 'dap-dlv-go)
+        ```
+    <!-- end list -->
+
+    1.  Usage
+
+        assume you have your code at \~/src/cool/cmd/app/app.go
+
+          - open your main package file e.g \~/src/cool/cmd/app/app.go
+          - or open a test file e.g app<sub>test</sub>.go
+          - add folder to lsp session where your go.mod is or would be
+              - M-x lsp-workspace-folders-add \~/src/cool
+          - set break point
+          - M-x dap-debug
+          - if you are debugging test files use "Go Dlv Launch File
+            Configuration"
+          - if you want to debug already running application select
+            "Go Dlv Attach Configuration"
+		  - if you want to debug remote application you need start
+            delve on remote machine first: `dlv --headless
+            --accept-multiclient attach 123 -l :1080`
```

Comment:
This can also be `dlv exec <program> --headless ...` or `dlv debug program --headless ...`. If you don't want to list multiple options, I think you should at least say "for example" and "see dlv usage doc for more command-line options".